packages/code-du-travail-frontend/src/common/Answer.js

```diff
@@ -28,12 +28,25 @@ const BackToResultsLink = ({ query }) => {
     </BacklinkContainer>
   );
 };
+const ROOT_CRUMB = {
+  title: "Thèmes",
+  slug: null
+};
+
 function getBreadcrumbs(items = []) {
-  return items.map(({ slug, label }) => (
-    <Link key={slug} route="themes" params={{ slug }}>
-      <a title={`Voir le theme ${label}`}>{label}</a>
-    </Link>
-  ));
+  if (!items.length) {
+    return [];
+  }
+  return [ROOT_CRUMB]
+    .concat(items.slice(0, -1))
+    .map(({ slug, title }) => (
+      <Link key={slug} route="themes" params={{ slug }}>
+        <a title={`Voir le theme ${title}`}>{title}</a>
+      </Link>
+    ))
+    .concat(
```

Comment:
to me the last bit should be a link too so we can navigate from an answer to its theme

Reply:
indeed